trunk/Bricksmith/Resources/Assets.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -0,0 +1,67 @@
+{
+  "images" : [
+    {
+      "filename" : "Bricksmith16.png",
+      "idiom" : "mac",
+      "scale" : "1x",
+      "size" : "16x16"
+    },
+    {
+      "filename" : "Bricksmith32-1.png",
+      "idiom" : "mac",
+      "scale" : "2x",
+      "size" : "16x16"
+    },
```

Comment:
All of these 2x images don't actually appear to be 2x. Since we don't actually have 2x assets at the moment, we shouldn't have 2x assets in the catalog.

Reply:
The 2x icons are just the 1x icons of the next size class.
I don't know what should be done differently for a specialized `2x` icon (seemed like they should work fine), but I can remove them if you would prefer.